Description: Don't warn about unvendored pydevd if it's the system version
 The (spurious) warning messages can cause problems.
Forwarded: not-needed
Origin: vendor
Author: Julian Gilbey <jdg@debian.org>

--- a/src/debugpy/_vendored/__init__.py
+++ b/src/debugpy/_vendored/__init__.py
@@ -90,6 +90,9 @@
     """Verify that only vendored modules have been imported."""
     if root is None:
         root = project_root(project)
+    # On Debian, the system version of pydevd is kept in sync with
+    # debugpy; we do not use a vendored version
+    pylibroot = "/usr/lib/python3/dist-packages/"
     extensions = []
     unvendored = {}
     for modname, mod in list(sys.modules.items()):
@@ -101,7 +104,7 @@
             filename = None
         if not filename:  # extension module
             extensions.append(modname)
-        elif not filename.startswith(root):
+        elif not filename.startswith(root) and not filename.startswith(pylibroot):
             unvendored[modname] = filename
     return unvendored, extensions
 
